ostack/deploy.sh

```diff
@@ -28,11 +37,10 @@ printf '%(%Y%m%d-%H:%M:%S)T\n'
 cd $DPL
 echo "cwd=$PWD"
 
-echo "PXD=$PXD"
+export PXD=PXD011124
 
 # Deploy cluster
-ansible-playbook --flush-cache -b --become-user=root \
-    $APP/ansible-slurm-cluster/playbook.yml -e "{'host_key_checking':false }"
+ansible-playbook --flush-cache $APP/ansible-slurm-cluster/playbook.yml -e "{'host_key_checking':false }"
```

Comment:
As you can see in in the [ansible doc on become](https://docs.ansible.com/ansible/2.4/become.html#command-line-options) this has a default value which is root.
So the only reliable way is to avoid:
```
name: play does not work in ECP
  hosts: localhost
  become: true
```

Reply:
I want to disable privilege escalation. If I run with _--become-user=root_ , It fails because execution with sudo. So I removed this param, it passes successfully with system user.